Rakefile

```diff
@@ -92,8 +92,18 @@ parsed_files = PARSER_FILES.map do |parser_file|
 end
 
 task "#{path}.gem" => package_parser_files
-desc "Generate all files used racc and kpeg"
-task :generate => parsed_files
+
+desc "Generate all files use racc and kpeg"
+task :generate do
+  unless ENV.key?('BUNDLE_GEMFILE')
+    Gem.install 'racc', '> 1.4.10'
+    Gem.install 'kpeg', '>= 1.3.3'
```

Comment:
@colby-swandale do you know if there's any rubygems API to better handle these dependencies without bundler?

Reply:
Which scenarios are you working with where Bundler would not be available?

Reply:
Bundler will be available, but this will be executed in 2 scenarios:
- When a developer runs `rake generate` (or any tasks that depend on it) manually
- When a user installs RDoc through git source

In the later case, I want to avoid installing all of RDoc's development dependencies as most of them aren't needed for generating the missing files.